plugins/sh/resources/META-INF/plugin.xml

```diff
@@ -16,8 +16,16 @@ Adds support for working with shell script files
 <li>Dedicated run/debug configuration for shell scripts</li>
 <li>Integration with external tools (<a href="https://github.com/koalaman/shellcheck">ShellCheck</a>, <a href="https://github.com/mvdan/sh">Shfmt</a>, <a href="https://explainshell.com/">Explainshell</a>)
 </li></ul>]]></description>
+
+  <extensionPoints>
+    <extensionPoint name="shSupport" interface="com.intellij.sh.ShSupport" dynamic="true"/>
+  </extensionPoints>
+
+  <extensions defaultExtensionNs="com.jetbrains.sh">
```

Comment:
If we use `qualifiedName` this block can be removed

Reply:
done